### DISCUSSION TITLE
Release notes dates

### DISCUSSION THREAD

**Discussion by hughrawlinson** [wagtail/wagtail#8687] [General]
Hey, just a quick suggestion - could the release notes pages have the release date on the page? I had a look at [this release](https://docs.wagtail.org/en/stable/releases/2.15.html?highlight=lts), but the release date wasn't obvious. It's important to know so that I have an understanding of how long the LTS applies for.

**Comment by gasman** [comment]
Good idea! We do put release dates in [the changelog](https://github.com/wagtail/wagtail/blob/main/CHANGELOG.txt), but that's not really the most visible place. I've now added this on [the just-released 3.0.1 page](https://docs.wagtail.org/en/stable/releases/3.0.1.html), and I'll open an issue for retrospectively adding it to the other pages, on the offchance that someone here at Wagtail Space feels like picking it up...

**Reply by gasman** [reply]
issue opened now - https://github.com/wagtail/wagtail/issues/8691

**Reply by hughrawlinson** [reply]
PR opened now 😄 https://github.com/wagtail/wagtail/pull/8692
